tests/13-report.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+	<testsuite tests="0" failures="0" time="0.000" name="package/name">
```

Comment:
Yes, that XML is about what I had in mind. The testcases for the failing packages would have an additional `<failure>` element with the error message. Using a map to store the error messages for each package SGTM.